content/blog/2019/nov/grpc-a-powerful-way-to-improve-golang-apis/index.md

```diff
@@ -0,0 +1,286 @@
+---
+title: 'GRPC: A powerful way to improve your Golang APIs'
+date: '2019-11-17T08:00:00Z'
+cover: './cover.jpg'
+---
+
+![Photo by Israel Palacio on Unsplash](cover.jpg)
+
+Web APIs are everywhere, with REST been one of the most popular ways to distribute it. With recent technologies, there are improved ways to implement it, GRPC been one of them.
```

Comment:
...with REST being one of the most popular ways to implement them ... there are better way to construct them, gRPC being one of them